concordium-contracts-common/src/schema_json.rs

```diff
@@ -930,6 +930,41 @@ fn write_bytes_from_json_schema_type<W: Write>(
     }
 }
 
+impl Fields {
+    pub fn field_to_json(&self) -> serde_json::Value {
```

Comment:
I think the name will be confusing if we later have a JSON format for the schema itself, so maybe add that this is a JSON template of the schema.
Also, since it is a method on Fields, I don't see the need to include it in the name.
```suggestion
    pub fn to_json_template(&self) -> serde_json::Value {
```